lib/setup/install.js

```diff
@@ -81,11 +83,12 @@ var avn = function() {
  *
  * @private
  * @function setup.install.~plugins
- * @return {Promise}
+ * @return {Promise} Promise that resolves when plugin installation is /
+ * complete and indicates if any action took place.
  */
 var plugins = function() {
-  return installedPlugins().then(function(plugins) {
-    return Promise.all(plugins.map(function(plugin) {
+  return installedPlugins().then(function(_plugins) {
+    return Promise.all(_plugins.map(function(plugin) {
```

Comment:
the no-underscore-dangle rule should be erroring here; can we pick a better name, like `installedPlugins`?